src/list-grid.scss

```diff
@@ -138,6 +142,34 @@ $block: #{$fd-namespace}-grid-list;
       @include fd-flex-vertical-center();
 
       padding: 0 1rem;
+
+      .#{$blockToolbar} {
+        &--actions {
+          margin-left: auto;
+
+          > * {
+            margin-left: 0.5rem;
+
+            @include fd-rtl() {
+              margin-left: 0;
+              margin-right: 0.5rem;
+            }
+          }
+
+          @include fd-rtl() {
+            margin-left: 0;
+            margin-right: auto;
+          }
+        }
+
+        .#{$item}-input,
+        &--actions,
+        &--extra-content {
+          @include fd-flex-vertical-center();
+
+          height: 100%;
+        }
+      }
```

Comment:
You can't do this, it's violating the rules for self-contained elements. You can't modify the properties of one component (Toolbar) inside another (List-grid). You need a modifier class.

Reply:
current implementation of grid-list is not using `fd-toolbar__spacer`, so custom classes were introduced. when tested using `fd-toolbar__spacer`, these extra classes are not needed. so removed it.